src/Npgsql/NpgsqlConnector.cs

```diff
@@ -1315,6 +1315,16 @@ internal void ClearTransaction()
         /// </summary>
         internal bool IsSecure { get; private set; }
 
+        /// <summary>
+        /// Returns whether SCRAM-SHA256 is being user for the connection
+        /// </summary>
+        internal bool IsScram { get; private set; }
```

Comment:
One thing I missed in my previous review - I don't think we should have these, since they are only ever used by test assertions. It should be OK to assume in the tests that the npgsql_tests_scram user has been properly configured in the environment - I'll make this change before merging.

Reply:
Deferring for now